components/contribution-flow/index.js

```diff
@@ -388,7 +402,15 @@ class ContributionFlow extends React.Component {
   /** Steps component callback  */
   onStepChange = async step => {
     this.setState({ showSignIn: false });
-    this.pushStepRoute(step.name);
+    if (this.props.paymentMethod === 'crypto') {
```

Comment:
What is this? Code comment please.

Reply:
I've added a code comment here, the payment method needs to be passed in order to create the order, this is normally done in the final step of the contribution flow, but here in the crypto flow the final step is not the step we create the order, we create the order before the final step, which is why this was added. 👍🏽